index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend:wght@400;500&family=Manrope&family=Noto+Color+Emoji&display=swap" rel="stylesheet">
+    <link rel = "stylesheet" href="index.css">
+    <script type="module" src = "index.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@1.27.0/build/global/luxon.min.js"></script>
+    <title>Awesome Books</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h3><a href = "index.html">Awesome Books</a></h3>
+          <ul>
+            <li><a id="book-link" href="#">List</a></li>
+            <li class = "line"><a id="add-link" href="#">Add Book</a></li>
+            <li class = "line"><a id="contact-link" href="#">Contact</a></li>          
+          </ul>
+      </nav>
+      <div id = "date"></div>
+    </header>
+    <main id = "hidden">
+      <section id = "booklist"></section>
+      </section>
+      <section id = "Add-Books">
+        <div class = "form-center">
```

Comment:
To follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Please let's use space-less equal signs. 👍 
    - Kindly use `<div class="striped">` not  `<div class = "striped">` 

Please correct all occurrences 🛠